File: Selenium/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager

service = webdriver.FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Page title : ", driver.title)
    username = driver.find_element(By.ID, "username")
    password = driver.find_element(By.ID, "password")

    username.send_keys("admin")
    password.send_keys("password")

    login = driver.find_element(By.XPATH, "(//button[@type='submit'])[1]")

    login.click()

    driver.close()

File: Selenium/Activities/Activity2.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args){

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://v1.training-support.net/selenium/login-form");

        System.out.println("The title of the page is : " + driver.getTitle());

        WebElement usernameTxtBox = driver.findElement(By.id("username"));

        usernameTxtBox.sendKeys("admin");

        WebElement passwordTxtBox = driver.findElement(By.id("password"));

        passwordTxtBox.sendKeys("password");

        WebElement logInBtn = driver.findElement(By.xpath("(//button[@type='submit'])[1]"));

        logInBtn.click();

        driver.close();
    }
}
